validation.go

```diff
@@ -51,7 +67,9 @@ type ValidationRequired struct {
 }
 
 func (v ValidationRequired) validate(value string) error {
-	// TBD
+	if value == "" {
```

Comment:
What if the string is all white spaces?

Reply:
I thought that the grcp request will take care of there for some reason.. but just tested it out and it doesn't :D 
I'll fix it.. thanks!